conbench/app/runs.py

```diff
@@ -20,13 +25,17 @@ def page(self, benchmarks, baseline_run, contender_run):
             title="Run",
             benchmarks=benchmarks,
             compare_runs_url=compare_runs_url,
+            form=form,
         )
 
     def get(self, run_id):
         if self.public_data_off():
             return self.redirect("app.login")
 
         contender_run, baseline_run = self.get_display_run(run_id), None
+        if contender_run is None:
```

Comment:
This code needs to be removed.

```
        if contender_run is None:
            self.flash("Error getting run.")
            return self.redirect("app.index")
```

As mentioned below, the run page should still render even if the run entry in the database doesn't exist yet.